src/firmware_new/boards/frankie_v1/constants.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+// This file contains all constants that are shared between our software (AI)
+// and firmware code. Since this needs to be compiled by both C and C++, everything
+// should be defined in a way that's compatible with C.
+//
+// NOTE: This file is here until its moved to the shared folder after the new firmware
+// is setup
+//
+
+// the IPv6 multicast address, only ff02 is important, the rest is random
```

Comment:
Why is ff02 important? Was there a reason ff02 was chosen specifically?

Reply:
Its because thats a valid multicast prefix: https://en.wikipedia.org/wiki/Solicited-node_multicast_address

added link